src/Root.cpp

```diff
@@ -677,9 +691,9 @@ void cRoot::AuthenticateUser(int a_ClientID, const AString & a_Name, const AStri
 
 
 
-int cRoot::GetTotalChunkCount(void)
+size_t cRoot::GetTotalChunkCount(void)
```

Comment:
This method is called in `WebAdmin.cpp:374`, changing the format type to `%zu`, fixes it.
`Printf(NumChunks, "%d", cRoot::Get()->GetTotalChunkCount());`